doc/test-framework/scorecard.md

```diff
@@ -325,11 +325,157 @@ Example of a valid JSON output:
+          ```
+    1. Create a `Secret` manifest `scorecard-secret.yaml` containing the operator's namespace (if any) the `Config`'s base64 encoding as a `spec.data` value under the key `kubeconfig`:
+          ```yaml
+          apiVersion: v1
+          kind: Secret
+          metadata:
+            name: scorecard-kubeconfig
+            namespace: ${your_namespace}
+          data:
+            kubeconfig: ${kubeconfig_base64}
+          ```
+    1. Apply the secret in-cluster:
+          ```sh
+          $ kubectl apply -f scorecard-secret.yaml
+          ```
+    1. Insert a volume referring to the `Secret` into the operator's `Deployment`:
+          ```yaml
+          spec:
+            install:
+              spec:
+                deployments:
+                - name: memcached-operator
+                  spec:
+                    ...
+                    template:
+                      ...
+                      spec:
+                        containers:
+                        ...
+                        volumes:
+                        # scorecard kubeconfig volume
+                        - name: scorecard-kubeconfig
+                          secret:
+                            secretName: scorecard-kubeconfig
+                            items:
+                            - key: kubeconfig
+                              path: config
+          ```
+1. Insert a volume mount and `KUBECONFIG` environment variable into each container in your operator's `Deployment`:
+    ```yaml
+    spec:
+      install:
+        spec:
+          deployments:
+          - name: memcached-operator
+            spec:
+              ...
+              template:
+                ...
+                spec:
+                  containers:
+                  - name: container1
+                    ...
+                    volumeMounts:
+                    # scorecard kubeconfig volume mount
+                    - name: scorecard-kubeconfig
+                      mountPath: /scorecard-secret
+                    env:
+                      # scorecard kubeconfig env
+                    - name: KUBECONFIG
+                      value: /scorecard-secret/config
+                  - name: container2
+                    # Do the same for this and all other containers.
+                    ...
+    ```
+1. Insert the scorecard proxy container into the operator's `Deployment`:
```

Comment:
The same here. IMHO it can be confuse to the user since they can try to look for a "deployment" file. I think is easier to explain to add it in the CSV file and/or operator.yaml